### PR TITLE
release kubekey v1.2.1-rc.1

### PR DESCRIPTION
**Pull request by 24sama**
### What does this PR do?
Release KubeKey v1.2.1-rc.1

### CODE REVIEW

**Comment by 24sama** [comment]
Hi @LinuxSuRen !
This is my first time to use `ks-version`. I don't know if this PR is correct. Please help me to review it.

/cc @LinuxSuRen

File: kubekey-v1.2.1-rc.1.yaml [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+  phase: ready
+  repositories:
+  - action: pre-release
+    address: https://github.com/kubesphere/kubekey
+    branch: release-1.2
+    name: kubekey
+    provider: github
+    version: v1.2.1-rc.1
+  secret:
+    name: linuxsuren-bot
```
Comment (LinuxSuRen) [inline kubekey-v1.2.1-rc.1.yaml]: @pixiake already created a secret `release-bot` under the namespace of `release`.

And the secret `linuxsuren-bot` should not have the permission to create a release of kubekey.

**Review by LinuxSuRen — CHANGES_REQUESTED** [review]
Thanks for your contribution. But please take a look at the secret part.

hi @pixiake, @24sama  By the way, please consider submitting an owner into the [OWNER file](https://github.com/kubesphere-sigs/ks-versions/blob/master/OWNERS).

I am the owner of this project. But I'm not going to merge any PRs of the kubekey.

[In future versions](https://github.com/kubesphere-sigs/ks-releaser/issues/43), it's possible to support putting different projects into different directories.

File: kubekey-v1.2.1-rc.1.yaml (hunk 1 above)
Comment (24sama) [reply kubekey-v1.2.1-rc.1.yaml]: Ok, done.

**Comment by pixiake** [comment]
> 

@LinuxSuRen Hi，Can @24sama  and I both be owner ?

**Comment by LinuxSuRen** [comment]
> > 
> 
> @LinuxSuRen Hi，Can @24sama and I both be owner ?

Of course.

**Review by LinuxSuRen — COMMENTED** [review]
/lgtm

**Pull request merged**